File: beginner/javascript/1064.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split('\n');

var i = 0, cont = 0, n, media = 0;
    
for(i = 0; i < 6; i++){
    n = parseFloat(valores.shift());
    if(n > 0){
        cont++;
        media += n;
    }
}

media = media / cont;

console.log(cont + " valores positivos");
console.log(media.toFixed(1));

File: beginner/c/1064.c
#include <stdio.h>
int main(){

    int i = 0, cont = 0;
    double n, media = 0;
    
    for(i = 0; i < 6; i++){
        scanf("%lf", &n);
        if(n > 0){
            cont++;
            media += n;
        }
    }

    media = media / cont;
    
    printf("%d valores positivos\n", cont);
    printf("%.1lf\n", media);

    return 0;
    
}